packer/scripts/modules/file-helper.psm1

```diff
@@ -24,7 +24,10 @@ function Get-FileFromInternet {
         return $output
     }
 
+    # Temporarily disabling progress bar for significantly faster downloads
```

Comment:
By "significant" I mean going from a download that would not complete in an hour to one that completed in something on the order of 10-20 minutes.